build.gradle

```diff
@@ -19,7 +19,7 @@ dependencies {
     compile 'org.glassfish.jersey.containers:jersey-container-servlet:2.25.1'
     compile 'org.glassfish.jersey.media:jersey-media-json-jackson:2.25.1'
     compile group: 'mysql', name: 'mysql-connector-java', version: '8.0.18'
-    testCompile group: 'com.h2database', name: 'h2', version: '1.3.148'
+    testCompile group: 'com.h2database', name: 'h2', version: '1.4.200'
```

Comment:
If you don't use it, don't add the dependency. After a while people forget about if and why it is needed...